### DISCUSSION TITLE
How do I start

### DISCUSSION THREAD

**Discussion by crystalshay2es** [crypto-org-chain/chain-main#934] [Q&A]
Hmm

**Comment by aw126** [comment]
Thanks for your interest in Crypto.org Chain! We recommend visiting our documentation at https://crypto.org/docs/chain-details/introduction.html to get started. The doc provides a comprehensive overview of our chain and its features, detailed guides to running nodes on the chain and more. 
Additionally, we encourage you to join our community on Discord at https://discord.gg/cryptoorg, which is a great resource for staying up-to-date with the latest developments and connecting with our dev community.
